app/Models/Context.php

```diff
@@ -242,37 +292,37 @@ public static function currentGet($array = false) {
 	}
 
 	/**
-	 * @return bool true if the current request targets all feeds (main view), false otherwise.
+	 * true if the current request targets all feeds (main view), false otherwise.
```

Comment:
Removing the `@return` instruction breaks the understanding of the PHPDocs.
Same comment than https://github.com/FreshRSS/FreshRSS/pull/5106#discussion_r1130735488